themes/default/content/registry/packages/aws-static-website/_index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: AWS Static Website
+meta_desc: Provides an overview of the AWS Static Website component.
+layout: overview
+---
+
+This component makes it easy to deploy a static website to s3 along with an optional CloudFront distribution using any of the supported Pulumi programming languages including markup languages like YAML and JSON.
+
+## Examples
+
+{{< chooser language "typescript,go,python,yaml" >}}
+{{% choosable language typescript %}}
+
+```typescript
+import * as staticwebsite from "@pulumi/aws-static-website"
+
+const args =  {
+    sitePath: "../website/build",
+} as staticwebsite.WebsiteArgs
+
+const site = new staticwebsite.Website("website", args);
+```
+
+{{% /choosable %}}
+{{% choosable language go %}}
+
+```go
+import (
+	website "github.com/pulumi/pulumi-aws-static-website/sdk/go/aws-static-website"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+
+		website, err := website.NewWebsite(ctx, "website", &website.WebsiteArgs{
+			SitePath: pulumi.String("../website/build"),
+		})
+		if err != nil {
+			return err
+		}
+
+		ctx.Export("websiteURL", website.WebsiteURL)
+		return nil
+	})
+}
+```
+
+{{% /choosable %}}
+{{% choosable language python %}}
+
+```python
+import pulumi
+from pulumi_aws_static_website import Website
+
+web = Website('website',
+    sitePath="../website/build")
+
+pulumi.export('websiteURL',  web.websiteURL)
+```
+
+{{% /choosable %}}
+{{% choosable language yaml %}}
+
+```yaml
+resources:
+  web:
+    type: "aws-static-website:index:Website"
+    properties:
+      sitePath: "../website/build"
+outputs:
+  websiteURL: ${web.websiteURL}
+```
+
+{{% /choosable %}}
+{{< /chooser >}}
+
+## Input Properties
+
+This component takes the following inputs.
+
+- sitePath - the root directory containing the website's contents to be served (required)
+- withCDN - provision a CloudFront CDN to serve content
+- targetDomain - the domain used to serve the content. A Route53 hosted zone must exist for this domain is this option is specified
+- index.html - the default document for the site. Defaults to index.html
+- error404 - the default 404 error page
+- certificateARN - the ARN of the ACM certificate to use for serving HTTPS. If one is not provided, a certificate will be created during the provisioning process
+- cacheTTL - TTL inseconds for cached objects
```

Comment:
Since the component doesn't invalidate the CloudFront cache, it'd be good to quite what the default is.